src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -445,6 +445,14 @@ public String getDisplayName() {
         }
 
         @Restricted(NoExternalUse.class)
+        public FormValidation doCheckIncludes(@QueryParameter String value) {
+            if (value.isEmpty()) {
+                return FormValidation.warning("Did you mean to use * to match all branches?");
```

Comment:
Nothing else in this class is internationalized yet either, but I can do it for this new code.